### ISSUE TITLE
Issues even after adding the Stage Owner as the Service Principal owner

### ISSUE THREAD

**Issue by LucX4** [MicrosoftDocs/power-platform#2181]
[Enter feedback here]

Hello Team,

I'm trying to follow this step:

![image](https://github.com/MicrosoftDocs/power-platform/assets/105860879/e2234d34-66f0-49bb-83a5-bf622643105f)

But, even after adding the stage owner as the App registration owner, I'm still getting the error message below:

![image](https://github.com/MicrosoftDocs/power-platform/assets/105860879/a889eaa1-52b3-4615-97fe-3ab5f12b480a)

Can you please advise?
Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0ba134b5-4cc6-a39f-471a-02830683c555
* Version Independent ID: 0ba134b5-4cc6-a39f-471a-02830683c555
* Content: [Deploy pipelines as a service principal or pipeline owner - Power Platform](https://learn.microsoft.com/en-us/power-platform/alm/delegated-deployments-setup)
* Content Source: [power-platform/alm/delegated-deployments-setup.md](https://github.com/MicrosoftDocs/power-platform/blob/main/power-platform/alm/delegated-deployments-setup.md)
* Service: **power-platform**
* Sub-service: **alm**
* GitHub Login: @caburk
* Microsoft Alias: **caburk**

**Comment by LucX4** [comment]
Hello guys, @caburk @phecke @KumarVivek
Thanks for any advice on this issue.

**Comment by bishalgoswami** [comment]
Thanks for your comment, @LucX4. This issue has been assigned to the doc author for review.

**Comment by FAC-1312** [comment]
Hello, I'm facing to the same error, any solution?

**Comment by LucX4** [comment]
When searching for the Client ID in Azure Portal, we see two different things: 'Application' and 'Service Principal'.
In our case, the Stage Owner was added as an owner only in the Application. After adding it on the 'Service Principle' page too, we were able to fix the issue.
![image](https://github.com/MicrosoftDocs/power-platform/assets/105860879/38625db8-86c2-4047-a762-33e06ec8d0c2)
@FAC-1312 @caburk @bishalgoswami FYI
